src/Avalonia.Base/Media/Imaging/PixelFormatWriter.cs

```diff
@@ -27,6 +27,24 @@ public void WriteNext(Rgba8888Pixel pixel)
         public void Reset(IntPtr address) => _address = (byte*)address;
     }
 
+    public unsafe struct Rgb32PixelFormatWriter : IPixelFormatWriter
+    {
+        private byte* _address;
+        public void WriteNext(Rgba8888Pixel pixel)
+        {
+            var address = _address;
+
+            address[0] = pixel.R;
+            address[1] = pixel.G;
+            address[2] = pixel.B;
+            address[3] = 255;
+
+            _address += 3;
```

Comment:
The increment should probably be 4 bytes.

```suggestion
            _address += 4;
```